lib/travis/build/templates/header.sh

```diff
@@ -52,9 +53,12 @@ travis_cmd() {
   fi
 
   if [[ -n "$retry" ]]; then
-    travis_retry eval "$cmd"
+    travis_retry eval "$cmd $secure"
   else
-    eval "$cmd"
+    eval "$cmd $secure"
+    if [[ -n $secure && $? -ne 0 ]]; then
+      echo "The previous command failed"
```

Comment:
https://github.com/travis-ci/travis-build/pull/939#issuecomment-274646429

` echo "The previous command failed, probably due to malformed secret. Please be sure to escape special characters."` 

does it log spaces as whitespace (for leading whitespace)?